src/components/Form/formStyling.tsx

```diff
@@ -25,6 +25,7 @@ export const StyledCard = styled.div`
     width: 100%;
     height: 80%;
     border-radius: 10px;
+    border: 1px solid ${(props) => props.theme.primaryForegroundColor};
```

Comment:
I would remove the border from the `StyledCard` after changing it to be a styled `Paper` component

Reply:
Agreed